Library/Homebrew/language/python.rb

```diff
@@ -365,7 +365,9 @@ def create(system_site_packages: true, without_pip: true)
         def pip_install(targets, build_isolation: true)
           targets = Array(targets)
           targets.each do |t|
-            if t.is_a?(Resource)
+            if t.is_a?(Resource) && t.url&.end_with?("-none-any.whl")
+              t.stage { do_install(Pathname.pwd/t.downloader.basename, build_isolation:) }
```

Comment:
```suggestion
              t.stage { do_install(t.cached_download, build_isolation:) }
```
This may work too. Not sure why we'd prefer one or the other though.

Reply:
`stage` does fetch + copy to the PWD whereas `cached_download` ignores whatever `unpack_strategy` does (which is probably nothing much)

Probably equivalent in this case though preferring PWD makes sense